package Exceptionhandling.java;

public class ExceptiobDemo {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		
//		int a=20;
//		int b=0;
		try 
		{
		int [] a =new int[5];
		}
		
		catch(Exception e )
		{
			System.out.println(e);
		}
		
		System.out.println("Exception handled sucessfully");
		}

}
